docker-compose.override.yml

```diff
@@ -5,10 +5,11 @@ services:
     volumes:
       - db-volume:/var/lib/mysql
   nginx:
+    container_name: 'wiki-nginx'
     volumes:
       - mediawiki-volume:/srv/mediawiki
-    ports:
-      - "${SERVER_PORT:-8080}:80"
+    networks:
+      nginx-network:
```

Comment:
I am not sure how this is backward compatible? I think we have to stop the wiki server. Implement this change, then bring the ingress up and restart service?

Reply:
Yeah actually I meant backwards compatible in a loose sense as this will not do much change to the way wiki works now.